<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="b_save">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Save path for results</string>
       </property>
       <property name="text">
        <string>Save Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_save">
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="b_vid">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Video you wish to analyse</string>
       </property>
       <property name="text">
        <string>Open Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_vid">
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="b_depth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Depth estimation running file</string>
       </property>
       <property name="text">
        <string>Depth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_depth">
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="b_of">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Optical flow running file</string>
       </property>
       <property name="text">
        <string>Optical flow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_of">
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="b_colour">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Filling colour around the video</string>
       </property>
       <property name="text">
        <string>Image Fill Colour</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_colour">
       <property name="font">
        <font>
         <family>GE Inspira</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="b_run">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>GE Inspira</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Start the calculation</string>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
